README.md

```diff
@@ -13,6 +13,7 @@ A set of Grafana dashboards and Prometheus alerts for Kubernetes.
 | release-0.2  | v1.14.1 and before | v2.11.0+  |
 | release-0.3  | v1.17 and before   | v2.11.0+  |
 | release-0.4  | v1.18              | v2.11.0+  |
+| release-0.5  | v1.18+             | v2.11.0+  |
```

Comment:
```suggestion
| release-0.5  | v1.19+             | v2.11.0+  |
```

Reply:
Or I don't see the point in this release?

Reply:
As discussed on Kubernetes slack we'll set `v1.19` and reserve `release-0.6` for `v1.19+`

Reply:
xref https://kubernetes.slack.com/archives/CAX9GU941/p1596108610030400